### ISSUE TITLE
output invalid code

### ISSUE THREAD

**Issue by lovetingyuan** [coderaiser/putout#133]
this is logger.ts
```javascript
const loop = () => {}

export const devLogger = (() => {
  if (process.env.NODE_ENV === 'development') {
    return console
  }
  // return { log: loop, warn: loop, error: loop }
})()

```

after run `npx putout --fix logger.ts`

output: 

```js
export const devLogger = {
  if (process.env.NODE_ENV === 'development') {
    return console;
  }
  // return { log: loop, warn: loop, error: loop }
};

```
It has syntax error.

**Comment by coderaiser** [comment]
What version of 🐊**Putout** do you use? Do you have any rules disabled?

**Comment by lovetingyuan** [comment]
> What version of 🐊**Putout** do you use? Do you have any rules disabled?

I just run `npx putout --fix logger.ts`, so the version should be the latest one, and there is no any extra config or rules.

**Comment by coderaiser** [comment]
Fixed with e0b3f0e, landed in 🐊**Putout** [v29.0.8](https://github.com/coderaiser/putout/releases/tag/v29.0.8) 🎉. Is it works for you?

**Comment by lovetingyuan** [comment]
no, the problem is still here.
```
export const devLogger = {
  if (process.env.NODE_ENV === 'development') {
    return console;
  }
  // return { log: loop, warn: loop, error: loop }
};

```

**Comment by coderaiser** [comment]
Fixed with 398618f 🎉. Please re-install 🐊**Putout**, is it works for you?

**Comment by lovetingyuan** [comment]
yes, it works, thank you!
